liquidcore/home/discover.py

```diff
@@ -0,0 +1,20 @@
+import requests
+from django.conf import settings
+from requests.exceptions import ConnectionError
+from json.decoder import JSONDecodeError
+
+def get_status():
+    if not settings.DISCOVERY_URL:
+        raise RuntimeError("settings.DISCOVERY_URL was not configured")
+    try:
+        response = requests.get(settings.DISCOVERY_URL).json()
+        response.setdefault("running", True)
+        return response
+    except ConnectionError:
+        return {"running": False, "error": "connection_error"}
+    except JSONDecoderError:
+        return {"running": False, "error": "json_error"}
```

Comment:
Just an user-readable error message, but you're right. This looks like unneeded complexity.